mlflow/models/evaluation/base.py

```diff
@@ -1077,26 +1084,27 @@ def scatter_plot(eval_df, builtin_metrics, artifacts_dir):
                                              :py:class:`mlflow.models.MetricThreshold` used for
                                              model validation. Each metric name must either be the
                                              name of a builtin metric or the name of a custom
-                                             metric defined in the``custom_metrics`` parameter.
+                                             metric defined in the ``custom_metrics`` parameter.
```

Comment:
@jerrylian-db Can we add some documentation to `mlflow.evaluate()` (outside of the params section) explaining how this API can be used for model validation?

https://github.com/mlflow/mlflow/pull/6419#discussion_r948655290

Reply:
I've included that in this PR. See the `Models.rst` changes and also the `examples/evaluation/evaluate_with_model_validation.py`